tests/test_repeating.js

```diff
@@ -93,40 +97,27 @@ test('times with invalid argument is ignored', async t => {
   const { statusCode } = await got('http://example.test/')
   t.is(statusCode, 200)
 
+  await assertRejects(
+    got('http://example.test/'),
+    Error,
+    'Nock: No match for request'
+  )
+
   scope.done()
 })
 
-test('isDone() must consider repeated responses', t => {
+test('isDone() must consider repeated responses', async t => {
   const scope = nock('http://example.test')
     .get('/')
     .times(2)
     .reply(204)
 
-  function makeRequest(callback) {
-    const req = http.request(
-      {
-        host: 'example.test',
-        path: '/',
-        port: 80,
-      },
-      function(res) {
-        t.equal(res.statusCode, 204)
-        res.on('end', callback)
-        // Streams start in 'paused' mode and must be started.
-        // See https://nodejs.org/api/stream.html#stream_class_stream_readable
-        res.resume()
-      }
-    )
-    req.end()
+  // eslint-disable-next-line no-unused-vars
+  for (const _ of Array(2)) {
+    t.is(scope.isDone(), false)
+    await got('http://example.test/')
   }
+  t.is(scope.isDone(), true)
 
-  t.notOk(scope.isDone(), 'should not be done before all requests')
-  makeRequest(function() {
-    t.notOk(scope.isDone(), 'should not yet be done after the first request')
-    makeRequest(function() {
-      t.ok(scope.isDone(), 'should be done after the two requests are made')
-      scope.done()
-      t.end()
-    })
-  })
+  scope.done()
```

Comment:
Is this last assertion needed considering the the `isDone` check just before?

Reply:
Probably not!